sdks/python/gen_protos.py

```diff
@@ -76,6 +77,14 @@ def generate_proto_files(force=False):
     try:
       from grpc_tools import protoc
     except ImportError:
+      if platform.system() == 'Windows':
+        # For Windows, grpcio-tools has to be installed manually.
+        logging.warning(
```

Comment:
Raise this as an exception. Otherwise the build will "succeed" and they'll get much harder to diagnose errors later.